pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -467,6 +468,28 @@ func (c *Cluster) EnableServiceMonitor(activeDaemon string) error {
 	}
 	serviceMonitor.Spec.NamespaceSelector.MatchNames = []string{c.clusterInfo.Namespace}
 	serviceMonitor.Spec.Selector.MatchLabels = c.selectorLabels(activeDaemon)
+
+	// If the cephcluster is managed by another resource,
+	// add the name of that resource as a label on the ceph metrics.
+	if c.spec.Labels != nil {
+		monitoringLabels, ok := c.spec.Labels["monitoring"]
+		if ok {
+			managedBy, ok := monitoringLabels["managedBy"]
+			if ok {
+				relabelConfig := monitoringv1.RelabelConfig{
+					TargetLabel: "managedBy",
+					Replacement: managedBy,
+				}
+				serviceMonitor.Spec.Endpoints[0].RelabelConfigs = append(
+					serviceMonitor.Spec.Endpoints[0].RelabelConfigs, &relabelConfig)
+			} else {
+				logger.Errorf("Could not find managedBy label in Monitoring labels.")
```

Comment:
These labels are completely optional, right? If so, let's just print info.
```suggestion
				logger.Info("Could not find managedBy label in Monitoring labels.")
```

Reply:
Yes, the labels are optional.